packages/css-data/src/parse-css-value.test.ts

```diff
@@ -374,3 +374,57 @@ test("parse unknown properties as unparsed", () => {
     }
   );
 });
+
+test("parses a valid translate value", () => {
+  expect(parseCssValue("scale", "1.5")).toEqual({
+    type: "tuple",
+    value: [
+      {
+        type: "keyword",
```

Comment:
why do you use keyword everywhere for numbers? We have unit: 'number' for such values.

Reply:
`none` is the only keyword here

Reply:
Done, i remember seeing it somewhere and just continued it everytime. Will use `number` from now on 👍